src/NuGet.Services.EndToEnd/LicenseTests.cs

```diff
@@ -39,6 +40,7 @@ public async Task PushedPackageWithLicenseExpressionIsAvailableInV3()
             Assert.All(packageRegistrationList, x => Assert.Equal(package.Properties.LicenseMetadata.License, x.CatalogEntry.LicenseExpression));
             Assert.All(packageRegistrationList, x => Assert.Equal(expectedPath, new Uri(x.CatalogEntry.LicenseUrl).AbsolutePath));
             await _clients.FlatContainer.WaitForPackageAsync(package.Id, package.NormalizedVersion, _logger);
+            VerifyLicenseUrlForV2V3Search(package, expectedPath);
```

Comment:
Consider adding a test for legacy license URL. This would be appropriate if we are not disabling license URL very soon. What is the time table for rejecting legacy license URL on gallery?

Reply:
That would be the question for @karann-msft. My understanding was "not soon".

---
In reply to: [252330017](https://github.com/NuGet/NuGet.Services.EndToEnd/pull/77#discussion_r252330017) [](ancestors = 252330017)